src/EntryPoint.cs

```diff
@@ -66,6 +66,7 @@ public static void Main (string[] args)
 							break;
 						}
 					}},
+				{ "github-pat=", "Sets the PAT required to access VSTS issues", v => options.GithubPAT = v},
```

Comment:
should say GitHub issues instead of VSTS issues

Reply:
Fixed in https://github.com/chamons/clio/pull/39